src/main/java/com/nha/abdm/wrapper/common/requests/SessionManager.java

```diff
@@ -52,6 +53,7 @@ private String fetchAccessToken() {
     return accessToken;
   }
 
+  @Scheduled(initialDelay = 18 * 60 * 1000, fixedRate = 18 * 60 * 1000)
```

Comment:
lets add a comment about the time like 18 minutes and why this is needed

Reply:
Added JavaDoc for better understanding.